docs/operator-manual/ingress.md

```diff
@@ -246,6 +246,13 @@ http {
     }
 }
 ```
+ALBs and Classic ELBs don't fully support HTTP2/gRPC, which is used by the `argocd` CLI.
```

Comment:
I think we should recommend using `--grpc-web-root-path` mainly because gRPC does not support a non-root path. Can you please change the description accordingly?